src/main/resources/templates/player/assignment/sequence/phase/evaluation/method/draxo/_draxo-show.html

```diff
@@ -15,65 +15,62 @@
                             <i class="red x icon" th:if="${optionType?.name == 'NEGATIVE'}"></i>
                             <i class="question icon" th:if="${optionType?.name == 'UNKNOWN'}"></i>
-
-                            <div class="content">
-                                <div class="title" th:text="${criteria}">R</div>
-                                <div class="description"
-                                     th:text="#{${criteria.getMessageI18nKey(T(org.elaastic.questions.assignment.sequence.peergrading.draxo.criteria.CriteriaMessageKey).Header)}}">
-                                    ...
-                                </div>
-                            </div>
                         </div>
-                    </th:block>
-                </div>
+                    </div>
+                </th:block>
+            </div>
 
-                <th:block th:with="rejectedCriteria = ${model.draxoEvaluation.getRejectedCriteria()}">
-                    <div class="ui form" th:if="${rejectedCriteria}">
-                        <div class="inline field">
-                            <label th:text="#{${rejectedCriteria.getMessageI18nKey(T(org.elaastic.questions.assignment.sequence.peergrading.draxo.criteria.CriteriaMessageKey).Question)}}+' :'"></label>
-                            <span th:text="#{${model.draxoEvaluation.get(rejectedCriteria).codeI18n}}"></span>
+            <th:block th:with="rejectedCriteria = ${model.draxoEvaluation.getRejectedCriteria()}">
+                <div class="comment-area" th:if="${rejectedCriteria}">
+                    <div class="comment-header">
+                        <h4 th:text="#{${rejectedCriteria.getMessageI18nKey(T(org.elaastic.questions.assignment.sequence.peergrading.draxo.criteria.CriteriaMessageKey).Question)}}+' :'">
+                            The answer is readable :</h4>
+                        <div class="ui label" th:text="#{${model.draxoEvaluation.get(rejectedCriteria).codeI18n}}">
+                            No
                         </div>
-
-                        <div class="field" th:if="${model.draxoEvaluation.getExplanation()}">
-                            <label th:text="#{draxo.feedback.label}">Feedback:</label>
-                            <p th:text="${model.draxoEvaluation.getExplanation()}"></p>
+                    </div>
+                    <div class="comment" th:if="${model.draxoEvaluation.getExplanation()}">
+                        <h4 th:text="#{draxo.feedback.label}">Feedback:</h4>
+                        <div class="ui label" th:text="${model.draxoEvaluation.getExplanation()}">
+                            I am a relevant and constructive comment, in order to explain why the answer provided above is not decipherable.
                         </div>
                     </div>
-                </th:block>
-            </td>
-        </tr>
-
-        </tbody>
-    </table>
-
-</div>
\ No newline at end of file
+                </div>
```

Comment:
The vertical separator is missing and I feel it's confusing without it.

Expected: 
![CleanShot 2024-04-18 at 10 33 42@2x](https://github.com/elaastic/elaastic-questions-server/assets/50367862/92d15585-011c-4fa1-97a5-142e6b297834)

Actual: 
![CleanShot 2024-04-18 at 10 34 30](https://github.com/elaastic/elaastic-questions-server/assets/50367862/f01cc05b-a252-428c-8f44-e433f9899199)

Reply:
![image](https://github.com/elaastic/elaastic-questions-server/assets/75416431/4aef7bea-5199-462f-9089-221dbb1e5b3a)